utilities/pleiades/launch-dask-scheduler.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#PBS -N dask-sched
+#PBS -W group_list=g26209
+#PBS -l select=1:ncpus=20:mpiprocs=5:model=ivy
+#PBS -l walltime=7:59:00
+#PBS -j oe
+#PBS -m abe
+
+export OMP_NUM_THREADS=4
+export LANG="en_US.utf8"
+export LANGUAGE="en_US.utf8"
+export LC_ALL="en_US.utf8"
+
+# Connect with
+# >>> from dask.distributed import Client
+# >>> client = Client(scheduler_file='~/scheduler.json')
+
+# Setup Environment
+module purge
+source activate pangeo
```

Comment:
Can this be done in the user's environment instead?  It seems odd to bake in the name of a particular conda environment.

Reply:
but how would we guarantee that the user's environment actually has dask?

Reply:
Presumably this would err if they didn't have dask.

Calling `source activate pangeo` doesn't guarantee this either though, unless they happen to have the same environment as you do.